lib/govuk_content_schemas/schema_combiner.rb

```diff
@@ -60,6 +74,20 @@ def add_format_field(schema)
     end
   end
 
+  # v2 schemas do not require these properties.
+  def update_required_properties(schema)
+    return if schemas.key?(:metadata) && schemas.key?(:base_links) # v1 schema
+
+    if schema['required']
```

Comment:
Fair point, given what @tijmenb has commented about `previous_version` I think we need a to define v1 and v2 properties separately, preserving common properties in `formats/metadata.json` and explicitly combine what's needed...?